joomla/joomla-cms
# [5.2] Static function getList deprecated in mod_stats

### Summary of Changes

Static function getList is after applying pull request deprecated and new non-static method getStats created in StatsHelper.


### Testing Instructions

1. Make sure you have site module mod_stats published
2. You should see module output on frontend
3. Deprecated static function getList should also work

![mod_stats](https://github.com/joomla/joomla-cms/assets/59166858/33eb0e83-f696-4278-9b17-5ed05a9db1e8)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
